tests/namespace/test_utilities_namespace.py

```diff
@@ -0,0 +1,87 @@
+import importlib
+import pathlib
+
+import pytest
+
+from pyvista.core.errors import PyVistaDeprecationWarning
+
+namespace_data = pathlib.Path(__file__).parent / 'namespace-utilities.txt'
+with open(namespace_data) as f:
+    namespace = f.read().splitlines()
+    # ignore commented data
+    namespace = [n.split(', ')[0] for n in namespace if not n.startswith('#')]
+
+
+@pytest.mark.parametrize('name', namespace)
+def test_utilities_namespace(name):
+    with pytest.warns(PyVistaDeprecationWarning):
+        utilities = importlib.import_module('pyvista.utilities')
```

Comment:
Do we need importlib in the tests in this module? Why not just
```suggestion
        from pyvista import utilities
```
here, and similarly later?

Reply:
Because of all the `*` imports, `from pyvista import utilities` is NOT equal to `import pyvista.utilities`. `from pyvista import utilities` gives us `pyvista.plotting.utilities` now.

re `importlib` vs `from ... import`, it doesn't matter either way

Reply:
Ah, I see. Sort of. I don't think star imports matter here. It's just that `import pyvista.utilities` forces a module import, whereas indeed the `from` import looks up the name in `pyvista/__init__.py`'s namespace.

So why not
```suggestion
        import pyvista.utilities as utilities
```
It's really the dynamic import that I'd like to avoid if possible.